.github/workflows/errors.yml

```diff
@@ -0,0 +1,55 @@
+on: push
+
+jobs:
+  verify:
+    name: "Verify errors"
+    runs-on: ubuntu-latest
+    steps:
+      # This step checks out a copy of your repository.
+      - uses: actions/checkout@v2
+
+      # run checks on a series of directories
+      - name: Trying to run in a non-existing directory should fail
+        uses: citation-file-format/cffconvert-github-action@tests
+        with:
+          FAILURE_EXPECTED: '1'
```

Comment:
Better to use Boolean (True|False) instead of '1' or '0'.
See:
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#literals

Reply:
But then what do you suggest the bash script does with those values?

Reply:
Sure, for the shell script it doesn't matter. But the action definition will be following the right syntax.

Reply:
I've now replaced it with string representations ``"true"`` and ``"false"``, since the arguments that I'm comparing are strings regardless.